src/rocks-command/rocks/commands/report/host/vm/config/__init__.py

```diff
@@ -194,6 +194,9 @@ def getInterfaces(self, node):
 			returnxml.append("    <interface type='bridge'>")
 			returnxml.append("      <source bridge='%s'/>" % bridged_inter["device"])
 			returnxml.append("      <mac address='%s'/>" % bridged_inter["mac"])
+			returnxml.append("      <filterref filter='clean-traffic'>")
```

Comment:
I commented on the bottom in general discussion, otherwise this thread is shown in the code and it's not clear what's happening in the code.